### PR TITLE
Changed recursive-include to global-include for *.pyx files.

### PR DESCRIPTION
**Pull request by cmccully**
I updated the MANIFEST.in file. I just tried to release a package, but none of the .pyx files were included. I realized that was because the *.pyx and *.pxd files were not being included correctly.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -5,7 +5,7 @@ include ez_setup.py
 include ah_bootstrap.py
 include setup.cfg
 
-recursive-include *.pyx *.c *.pxd
+global-include *.pyx *.c *.pxd
```
Comment (embray) [inline MANIFEST.in]: I think the problem here is this really should be `recursive-include <name of package> *.pyx *.c *.pxd`

**Comment by cmccully** [comment]
@embray That is a fair point about the recursive include. However, how do you recommend updating this? I would prefer to not have to have package maintainers updating the MANIFEST.in file.

**Comment by embray** [comment]
Unfortunately for better or worse (I'd say the latter) editing MANIFEST.in is the correct way to control what is and isn't included in the source dist of a Python project packaged with distutils.  I don't think the point of this package-template is that people who use it should have to learn *nothing* about Python packaging.  It just gives you a lot of boilerplate and instructions for editing it.

I do have an open issue at astropy/astropy-helpers#53 to address this specifically in the case of Cython modules, though I've also thought of adding a way *other* than MANIFEST.in to add files for a source dist that isn't so disjoint.

**Comment by cmccully** [comment]
I think I would still advocate the global-include for *.pyx, *.c, *.pxd, and *.h files. I can't think of a typical use case that you would not want to include all of these files in the package. If there was something you wanted to exclude, that could be done explicitly. What would the rationale be for doing a recursive include instead? In principle, we could do that, but we also need to update the astropy affiliated package documentation if we do.

**Comment by embray** [comment]
Yeah, maybe that's fine as an intermediate measure.  Ideally the sdist command itself really should know about these files implicitly--especially if they're listed as sources in an `Extension` (maybe .h files are a little trickier in this case since they aren't typically listed, but they should probably be picked up automatically anyways).

**Comment by bsipocz** [comment]
We're moving over to cookicutter, so I'm closing old unresolved PRs. Please reopen against the cookicutter branch if you feel this is still relevant.

**Pull request closed without merging**